azure-quantum/tests/unit/recordings/test_job_submit_honeywell.yaml

```diff
@@ -215,116 +215,24 @@ interactions:
-        "inputDataUri": "https://mystorage.blob.core.windows.net/job-00000000-0000-0000-0000-000000000000/inputData?sv=PLACEHOLDER&sr=b&sig=PLACEHOLDER&se=PLACEHOLDER&sp=r&rscd=attachment%3B%20filename%3Dhoneywell-job-00000000-0000-0000-0000-000000000000.input.json",
-        "inputDataFormat": "honeywell.openqasm.v1", "inputParams": {}, "providerId":
-        "honeywell", "target": "honeywell.hqs-lt-s1-apival", "metadata": null, "name":
-        "honeywell-job", "id": "00000000-0000-0000-0000-000000000000", "status": "Succeeded",
-        "outputDataFormat": "honeywell.quantum-results.v1", "outputDataUri": "https://mystorage.blob.core.windows.net/job-00000000-0000-0000-0000-000000000000/rawOutputData?sv=PLACEHOLDER&sr=b&sig=PLACEHOLDER&se=PLACEHOLDER&sp=r&rscd=attachment%3B%20filename%3Dhoneywell-job-00000000-0000-0000-0000-000000000000.output.json",
-        "creationTime": "2022-02-19T01:11:32.0566064+00:00", "beginExecutionTime":
-        "2022-02-19T01:11:35.253241+00:00", "endExecutionTime": "2022-02-19T01:11:35.253581+00:00",
-        "cancellationTime": null, "costEstimate": null, "errorData": null, "isCancelling":
-        false, "tags": [], "access_token": "fake_token"}'
-    headers:
-      content-length:
-      - '1410'
+      - '128'
       content-type:
       - application/json; charset=utf-8
-      transfer-encoding:
-      - chunked
-    status:
-      code: 200
-      message: OK
-- request:
-    body: null
-    headers:
-      Accept:
-      - application/xml
-      Accept-Encoding:
-      - gzip, deflate, br
-      Connection:
-      - keep-alive
-      User-Agent:
-      - azsdk-python-storage-blob/12.9.0 Python/3.7.10 (Darwin-21.3.0-x86_64-i386-64bit)
-      x-ms-date:
-      - Sat, 19 Feb 2022 01:11:45 GMT
-      x-ms-range:
-      - bytes=0-33554431
-      x-ms-version:
-      - '2020-10-02'
-    method: GET
-    uri: https://mystorage.blob.core.windows.net/job-00000000-0000-0000-0000-000000000000/rawOutputData?sv=PLACEHOLDER&sr=b&sig=PLACEHOLDER&se=PLACEHOLDER&sp=r&rscd=attachment%3B%20filename%3Dhoneywell-job-00000000-0000-0000-0000-000000000000.output.json
-  response:
-    body:
-      string: '{"c0": ["0"], "c1": ["0"], "c2": ["0"], "access_token": "fake_token"}'
-    headers:
-      accept-ranges:
-      - bytes
-      content-length:
-      - '39'
-      content-range:
-      - bytes 0-38/39
-      content-type:
-      - application/octet-stream
-      x-ms-blob-content-md5:
-      - G6E9ivtAth12WMpLNitMGA==
-      x-ms-blob-type:
-      - BlockBlob
-      x-ms-creation-time:
-      - Sat, 19 Feb 2022 01:11:34 GMT
-      x-ms-lease-state:
-      - available
-      x-ms-lease-status:
-      - unlocked
-      x-ms-server-encrypted:
-      - 'true'
-      x-ms-version:
-      - '2020-10-02'
     status:
-      code: 206
-      message: Partial Content
+      code: 403
```

Comment:
Looks like this job failed. 
The test should have probably failed or at least report a warning. This should run again vs a valid workspace.

Reply:
Oh, this should have been caught by the unit tests asserts, but unfortunately there was [some logic to suppress the error of not having the provider in the workspace and make it just a warning](https://github.com/microsoft/qdk-python/blob/179fb04f7b2ec5b87b26551722400f0dd437a575/azure-quantum/tests/unit/test_target.py#L235).
I will remove the logic to let the test fail in those cases.